File: py/boost-python/student.cpp
#include <iostream>  
#include <string>  
using namespace std;  
  
class student  
{  
public:  
    void setname(string str)  
    {  
        name_ = str;  
    }  
  
    string getname()  
    {  
        return name_;  
    }  
      
    void setage(int age)  
    {  
        age_ = age;  
    }  
      
    int getage()  
    {  
        return age_;  
    }  
      
private:  
    string name_;  
    int age_;  
  
};

File: java/dto/Student.java
/**
 * @author libo
 * @date 2018年9月20日 下午8:44:33
 */
public class Student {
    
    private Integer id;
    private String name;
    private Integer age;
    private String address;
    
    public Integer getId() {
        return id;
    }
    public void setId(Integer id) {
        this.id = id;
    }
    public String getName() {
        return name;
    }
    public void setName(String name) {
        this.name = name;
    }
    public Integer getAge() {
        return age;
    }
    public void setAge(Integer age) {
        this.age = age;
    }
    public String getAddress() {
        return address;
    }
    public void setAddress(String address) {
        this.address = address;
    }
    @Override
    public String toString() {
        return "Student [id=" + id + ", name=" + name + ", age=" + age + ", address=" + address + "]";
    }
    
    
}
